.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: rust
+os: osx
```

Comment:
Interesting!

Reply:
Yeah we need that for lipo, but like in said in https://github.com/mozilla/application-services/pull/6#issuecomment-389378704 we don't really need these lipoed libraries.